### ISSUE TITLE
Join Us - Mentors link

### ISSUE THREAD

**Issue by sheraaron** [carpentries/carpentries.org#104]
The "Mentors" link on the join us page under "We do this through" does not direct you to the mentoring page. It doesn't go anywhere.

**Comment by weaverbel** [comment]
Pinging @ErinBecker - where should this link to?

**Comment by ErinBecker** [comment]
Good catch @sheraaron! It is meant to be a direct in-page link to the Mentoring section in that same page (not an external mentoring page).

**Comment by ErinBecker** [comment]
I see - it needs to be changed to https://carpentries.org/community/#mentoring
